schema/defs.yaml

```diff
@@ -313,3 +314,21 @@ description: Common schema definitions shared by all themes
         moving along the line from beginning to end
       properties:
         side: { "$ref": "#/$defs/propertyDefinitions/side" }
+    cartographyContainer:
+      title: cartography
+      description: defines cartograhic hints for optimal use of Overture features in map making
```

Comment:
Cool.

I know it seems nitty, but would you mind punctuating the descriptions into classic sentence case (starting capital, ending full stop)?